src/PackagePublisher/PackagePublisher.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>PackagePublisher</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <description>Publishes packages to a feed, with retries.</description>
+  </metadata>
+  <files>
+    <file src="net451\" target="/" />
```

Comment:
I wondered the same but left it the same as SplitPackages. I will send out a separate PR for SplitPackage changes with netcoreapp1.0 only.